addons/account/wizard/accrued_orders.py

```diff
@@ -182,10 +182,10 @@ def _ellipsis(string, size):
                 for order_line in lines:
                     if is_purchase:
                         account = order_line.product_id.property_account_expense_id or order_line.product_id.categ_id.property_account_expense_categ_id
-                        amount = self.company_id.currency_id.round(order_line.qty_to_invoice * order_line.price_unit / rate)
-                        amount_currency = order_line.currency_id.round(order_line.qty_to_invoice * order_line.price_unit)
+                        amount = self.company_id.currency_id.round(order_line.qty_to_invoice * order_line.net_price_unit / rate)
```

Comment:
explain a bit more than [IMP] purchase: code improvement in the commit message. What was the purpose of the change